### PR TITLE
chore: porting the tutorial from branch proposal_v2.0 to main

### PR DESCRIPTION
**Pull request by evshary**
The Version 2.0 tutorial on the documentation website is still the same as Version 1.0.
I've followed the tutorial in the proposal_v2.0 branch and it works well, so I think it'll be great if we can put it on the website.
Every new developer who is interested in the Open AD Kit can try the latest version.
If there are any issues with the tutorial, we can also correct them fast.

I didn't join the discussion of the website before, so if there are any concerns about this, please let me know.

### CODE REVIEW

**Comment by ghost** [comment]
@evshary Thank you for your dedication! If you can add label "documentation" to this PR, the GitHub page will be created for your PR.

@ito-san I would appreciate it if you could review this PR. This PR seems to be derived from [your proposal branch](https://github.com/autowarefoundation/open-ad-kit-docs/tree/proposal_v2.0/docs).

**Comment by evshary** [comment]
> @evshary Thank you for your dedication! If you can add label "documentation" to this PR, the GitHub page will be created for your PR.
> 
> @ito-san I would appreciate it if you could review this PR. This PR seems to be derived from [your proposal branch](https://github.com/autowarefoundation/open-ad-kit-docs/tree/proposal_v2.0/docs).

@d-hashimoto Thanks for your reply. It seems like I don't have the privilege to add label. I'll ask for help.

**Comment by evshary** [comment]
I'm just aware that most of the commits in proposal_v2.0 are contributed by @ito-san.
Therefore I've updated the author in the git commit for giving the correct credit.
If you prefer to merge proposal_v2.0 to the main branch directly, please let me know. I can close this PR.
It's really convenient to have OADK 2.0 on the website, and I'm happy to make it happen sooner.

**Comment by ito-san** [comment]
@evshary Amazing, This is a great PR made by you! :clap:

File: docs/version-2.0/start-guide/installation/advanced-setup-setup-ava.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+1. Copy `meta-ewaol-ext` directory to the project root.
+
+   ```console
+   cp -rf meta-ewaol-machine/meta-ewaol-ext ~/meta-adlink-ampere
+   ```
+
+1. Download a missing patch in `meta-ewaol-ext` layer.
+
+   ```console
+   wget -P ~/meta-adlink-ampere/meta-ewaol-ext/recipes-ewaol/recipes-container/nvidia-container-toolkit/files https://raw.githubusercontent.com/autowarefoundation/open-ad-kit-docs/proposal_v2.0/docs/start-guide/installation/patch/0002-TEMPORARY-force-aarch64-rpm.patch
```
Comment (ito-san) [inline docs/version-2.0/start-guide/installation/advanced-setup-setup-ava.md]: Please do not use `proposal_v2.0` and correct the path to the file.

File: docs/version-2.0/start-guide/installation/system-setup-ava.md [hunk 2]
```diff
@@ -4,138 +4,139 @@
...
-The docker image of Autoware.Auto is registered in [GitLab Container Registry](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/container_registry/2511358).
-![GitLab Container Registry](images/system-setup-ava/gitlab-cr.png)
+Autoware.Universe runs as k3s clusters in Open AD Kit, so please download kubernetes yaml files to deploy Autoware to AVA platform.
 
-1. Copy docker image to AVA platform.
+1. Download.
 
    ```console
-   docker pull registry.gitlab.com/autowarefoundation/autoware.auto/autowareauto/arm64/openadkit-foxy:latest
+   wget https://raw.githubusercontent.com/autowarefoundation/open-ad-kit-docs/proposal_v2.0/docs/start-guide/installation/deployments/comhpc-deployments.zip
```
Comment (ito-san) [inline docs/version-2.0/start-guide/installation/system-setup-ava.md]: Please do not use `proposal_v2.0` and correct the path to the file.

File: docs/version-2.0/start-guide/installation/system-setup-ava.md [hunk 3]
```diff
@@ -4,138 +4,139 @@
...
-      NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
-      sda           8:16   1  28.9G  0 disk
-      `-sda1        8:17   1  28.9G  0 part
-      nvme0n1     259:0    0 119.2G  0 disk
-      |-nvme0n1p1 259:1    0   256M  0 part
-      |-nvme0n1p2 259:2    0   118G  0 part /
-      `-nvme0n1p3 259:3    0     1G  0 part
-      ```
+   ```console
+   wget -P /etc/sysctl.d https://raw.githubusercontent.com/autowarefoundation/open-ad-kit-docs/proposal_v2.0/docs/start-guide/installation/sysctl.d/60_cyclonedds.conf
```
Comment (ito-san) [inline docs/version-2.0/start-guide/installation/system-setup-ava.md]: Please do not use `proposal_v2.0` and correct the path to the file.

File: docs/version-2.0/start-guide/installation/system-setup-ava.md [hunk 4]
```diff
@@ -4,138 +4,139 @@
...
+In this test, we are using Cyclone DDS, so you also need to download configuration file of Cyclone DDS.
 
-   1. Update kernel parameters.
+1. Download `cyclonedds.xml`.
 
-      ```console
-      sysctl -p /etc/sysctl.d/60_cyclonedds.conf
-      ```
+   ```console
+   wget -P ~/cyclonedds https://raw.githubusercontent.com/autowarefoundation/open-ad-kit-docs/proposal_v2.0/docs/start-guide/installation/cyclonedds/cyclonedds.xml
```
Comment (ito-san) [inline docs/version-2.0/start-guide/installation/system-setup-ava.md]: Please do not use `proposal_v2.0` and correct the path to the file.

File: docs/version-2.0/start-guide/installation/system-setup-host.md [hunk 5]
```diff
@@ -4,46 +4,71 @@
...
+
+## Download **kernel configuration** file for tuning kernel parameters
+
+We have to reconfigure kernel parameters by using `sysctl` for system stability.
+
+1. Download.
 
    ```console
-   cp 60_cyclonedds.conf /etc/sysctl.d
+   wget -P /etc/sysctl.d https://raw.githubusercontent.com/autowarefoundation/open-ad-kit-docs/proposal_v2.0/docs/start-guide/installation/sysctl.d/60_cyclonedds.conf
```
Comment (ito-san) [inline docs/version-2.0/start-guide/installation/system-setup-host.md]: Please do not use `proposal_v2.0` and correct the path to the file.

File: docs/version-2.0/start-guide/installation/system-setup-host.md [hunk 6]
```diff
@@ -52,6 +77,16 @@ The docker image of scenario simulator is registered in [docker hub](https://hub
...
    ```
 
+## Download configuration file of Cyclone DDS
+
+In this test, we are using Cyclone DDS, so you also need to download configuration file of Cyclone DDS.
+
+1. Download `cyclonedds.xml`.
+
+   ```console
+   wget -P ~/cyclonedds https://raw.githubusercontent.com/autowarefoundation/open-ad-kit-docs/proposal_v2.0/docs/start-guide/installation/cyclonedds/cyclonedds.xml
```
Comment (ito-san) [inline docs/version-2.0/start-guide/installation/system-setup-host.md]: Please do not use `proposal_v2.0` and correct the path to the file.

**Review by ito-san — CHANGES_REQUESTED** [review]

**Comment by evshary** [comment]
@ito-san Thank you! I've already fixed the links.

**Review by ito-san — APPROVED** [review]
@evshary Thank you for the fix! LGTM.

**Pull request merged**